tests/all/unit/api/helpers/test_storage.py

```diff
@@ -32,3 +33,17 @@ def test_https_url(self):
         self.assertEqual(
             expected_file_url, create_url(request_url, '/some/path/image.png')
             )
+
+    def test_create_url(self):
+        """Method to test a generated url of uploaded file"""
+        request_url = "https://localhost:4200"
+        expected_file_url = 'https://localhost:4200/some/path/image.png'
+        self.assertEqual(
+            expected_file_url, create_url(request_url, '/some/path/image.png')
+            )
+
+    def test_generate_hash(self):
+        """Method to test generation of hash with key"""
+        with app.test_request_context():
```

Comment:
Remove this, this is a unit test

Reply:
Isn't this test is a part of unit test?

Reply:
This bootstraps the flask app. Unit Test by definition is pure function not depending on framework/database/network

Reply:
@iamareebjamal Got it.